app/views/publishers/organisations/_vacancy_awaiting_feedback.html.slim

```diff
@@ -10,11 +12,18 @@
           span class="card-component__item-label govuk-!-font-weight-bold" = t("jobs.manage.closing_date")
           = vacancy.application_deadline
 
-    dd.card-component__body class="vacancy__feedback"
-      ul.govuk-list
-        li class="govuk-!-width-full-width" = select_tag "vacancy[hired_status]", options_for_select(hired_status_options, nil), form: vacancy.id, include_blank: "--", class: "govuk-select form-control form-control-block govuk-!-width-full"
-        li class="govuk-!-width-full-width" = select_tag "vacancy[listed_elsewhere]", options_for_select(listed_elsewhere_options, nil), form: vacancy.id, include_blank: "--", class: "govuk-select form-control form-control-block govuk-!-width-full"
+    dd.card-component__body
+      = f.govuk_collection_select :hired_status,
+        hired_status_options,
+        :last,
+        :first,
+        html_options: { class: "govuk-!-width-full", form: vacancy.id }
+      = f.govuk_collection_select :listed_elsewhere,
+        listed_elsewhere_options,
+        :last,
+        :first,
+        html_options: { class: "govuk-!-width-full", form: vacancy.id }
 
     dd.card-component__actions
       ul.actions.govuk-list
-        li = submit_tag t("buttons.submit"), form: vacancy.id, class: "govuk-button govuk-!-margin-bottom-0"
+        li = submit_tag t("buttons.submit"), form: vacancy.id, class: "govuk-button govuk-!-margin-bottom-0 govuk-!-margin-top-6"
```

Comment:
what happens if you use `f.govuk_submit`?

Reply:
You can't add the form attribute, which is needed to disambiguate which form is being submitted.

Reply:
Opened an issue https://github.com/DFE-Digital/govuk_design_system_formbuilder/issues/252